Automatic Validation ended with: 
> 2026-02-12T16:42:56.6640153Z ##[error] Manifest Error: Duplicate installer entry found.

(Automated response - build 1221.)

--- a/manifests/d/DCSS/DungeonCrawlStoneSoup/0.34.0/DCSS.DungeonCrawlStoneSoup.installer.yaml
+++ b/manifests/d/DCSS/DungeonCrawlStoneSoup/0.34.0/DCSS.DungeonCrawlStoneSoup.installer.yaml
@@ -7,7 +7,6 @@ InstallerLocale: en-US
 Platform:
 - Windows.Desktop
 InstallerType: nullsoft
-Scope: machine
 InstallModes:
 - interactive
 - silent
@@ -20,14 +19,17 @@ InstallationMetadata:
   DefaultInstallLocation: '%ProgramFiles(x86)%\Crawl'
 Installers:
 - Architecture: x86
+  Scope: machine
   InstallerUrl: https://github.com/crawl/crawl/releases/download/0.34.0/dcss-0.34.0-win32-installer.exe
   InstallerSha256: 8D72BCBBE1C39A8B4FA171A922DEACF2BE4FF365DF99AEECCB702AE236D146C6
   InstallerSwitches:
     Custom: /ALLUSERS
 - Architecture: x86
+  Scope: user
   InstallerUrl: https://github.com/crawl/crawl/releases/download/0.34.0/dcss-0.34.0-win32-installer.exe
   InstallerSha256: 8D72BCBBE1C39A8B4FA171A922DEACF2BE4FF365DF99AEECCB702AE236D146C6
   InstallerSwitches:
     Custom: /CURRENTUSER
 ManifestType: installer
 ManifestVersion: 1.12.0
+

